test/models/company_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class CompanyTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
if you add the associations or validations, add some specs here please.

Reply:
Done